sphinx/tutorial/cylc/runtime/runtime-configuration.rst

```diff
@@ -0,0 +1,456 @@
+.. _DataPoint: https://www.metoffice.gov.uk/datapoint
+
+.. _tutorial-cylc-runtime-configuration:
+
+.. include:: ../../../hyperlinks.rst
+  :start-line: 1
+
+Runtime Configuration
+=====================
+
+
+In the last section we associated tasks with scripts and ran a simple suite. In
+this section we will look at how we can configure these tasks.
+
+
+Environment Variables
+---------------------
+
+We can specify environment variables in a task's ``[environment]`` section.
+These environment variables are then provided to :term:`jobs <job>` when they
+run.
+
+.. code-block:: cylc
+
+   [runtime]
+       [[countdown]]
+           script = seq $START_NUMBER
+           [[[environment]]]
+               START_NUMBER = 5
+
+Each job is also provided with some standard environment variables e.g:
+
+``CYLC_SUITE_RUN_DIR``
+    The path to the suite's :term:`run directory`
+    *(e.g. ~/cylc-run/suite)*.
+``CYLC_TASK_WORK_DIR``
+    The path to a task's :term:`work directory`
+    *(e.g. run-directory/work/cycle/task)*.
+``CYLC_TASK_CYCLE_POINT``
+    The :term:`cycle point` for a task
+    *(e.g. 20171009T0950)*.
+
+There are many more environment variables, see the `cylc user guide`_ for more
+information.
+
+
+Job Submission
+--------------
+
+By default cylc runs :term:`jobs <job>` on the machine where the suite is
+running. We can tell cylc to run jobs on other machines by setting the
+``[remote]host`` setting to the name of the host. E.G To run a task on the host
+``computehost`` you might write:
+
+.. code-block:: cylc
+
+   [runtime]
+       [[hello_computehost]]
+           script = echo "Hello Compute Host"
+           [[[remote]]]
+               host = computehost
+
+.. _background processes: https://en.wikipedia.org/wiki/Background_process
+.. _job scheduler: https://en.wikipedia.org/wiki/Job_scheduler
+
+.. _tutorial-batch-system:
+
+By default cylc runs executes jobs as `background processes`_.
+When we are running jobs on other compute hosts we will often want to
+use a :term:`batch system` (`job scheduler`_) to submit our job.
+Cylc supports the following :term:`batch systems <batch system>`:
+
+* loadleveler
+* lsf
+* pbs
+* sge
+* slurm
+
```

Comment:
It supports more batch schedulers than these listed according to the cylc docs. (Although whether you need to list all of them here is another question). Could link to the cylc docs for brevity. https://cylc.github.io/cylc/html/single/cug-html.html#A.5.1